randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -122,7 +122,8 @@ Extra - room_id: [3]
   > Door to Nocturnal Access
       Any of the following:
           Can Kill Super Geron
-          Speed Booster and Shinespark Tricks (Advanced) and Disabled Door Lock Rando and Disabled Entrance Rando
+          # Shinespark from Nav Room: https://youtu.be/4vs0MvX4oew
```

Comment:
preferably I would want to show the Geron actually getting destroyed

Reply:
Idk why but I never killed the Geron and it was gone (Maybe it was killing Mega Core-X???)

Reply:
made it a todo, someone can come back if they want to redo it (probably good to have less beams too?)
also added varia core-x requirement